### ISSUE TITLE
[1.2.0 Beta1] Tiny QA issues

### ISSUE THREAD

**Issue by jpobeda** [Cacti/cacti#2094]
Hi folks

Reporting some oddities found

1.- Seems like a dummy extra column in there

![firefox_2018-10-20_11-25-01](https://user-images.githubusercontent.com/13456559/47250853-d7bc9780-d485-11e8-9528-ae5d7898dc54.png)

2.- Values weren't passed, look up those `%s`

![firefox_2018-10-20_14-26-24](https://user-images.githubusercontent.com/13456559/47250858-eacf6780-d485-11e8-93f1-181161950ce3.png)

3. - I believe this one is from the Console -> Import/Export -> Import -> Data Source Profile menu
Not available for translation even after running `build_gettext.sh`

```console
./include/global_form.php:1640
```

![firefox_2018-10-20_15-42-25](https://user-images.githubusercontent.com/13456559/47250866-1d796000-d486-11e8-9f47-69af5484cbe7.png)

4.- So I got the warning and went to fix them, after the refresh they all went green but the doted square with the examples just got shrank there.

![firefox_2018-10-20_16-23-25](https://user-images.githubusercontent.com/13456559/47250888-a55f6a00-d486-11e8-9937-d340da6aab08.png)

5.- `build_gettext.sh` script duplicates files -> maybe for @ronytomen 

```console
[root@cacti12 locales]# ls -ltr po/
total 10708
-rw-r--r--. 1 cacti apache 443701 Oct 20 10:39 swedish_sweden.po~
-rw-r--r--. 1 cacti apache 457496 Oct 20 10:39 russian_russia.po~
-rw-r--r--. 1 cacti apache 468283 Oct 20 10:39 japanese_japan.po~
-rw-r--r--. 1 cacti apache 620459 Oct 20 10:39 german_germany.po~
-rw-r--r--. 1 cacti apache 483766 Oct 20 10:39 french_france.po~
-rw-r--r--. 1 cacti apache 554670 Oct 20 10:39 dutch_netherlands.po~
-rw-r--r--. 1 cacti apache 626257 Oct 20 10:39 chinese_china_simplified.po~
-rw-r--r--. 1 cacti apache 460090 Oct 20 10:39 bulgarian_bulgaria.po~
-rw-r--r--. 1 cacti apache 725525 Oct 20 10:39 spanish_spain.orig
-rw-r--r--. 1 root  root   762637 Oct 20 15:58 spanish_spain.po~
-rw-r--r--. 1 cacti apache 425178 Oct 20 16:43 cacti.pot
-rw-r--r--. 1 cacti apache 462355 Oct 20 16:43 bulgarian_bulgaria.po
-rw-r--r--. 1 cacti apache 628912 Oct 20 16:43 chinese_china_simplified.po
-rw-r--r--. 1 cacti apache 557028 Oct 20 16:43 dutch_netherlands.po
-rw-r--r--. 1 cacti apache 486194 Oct 20 16:43 french_france.po
-rw-r--r--. 1 cacti apache 622684 Oct 20 16:43 german_germany.po
-rw-r--r--. 1 cacti apache 470630 Oct 20 16:43 japanese_japan.po
-rw-r--r--. 1 cacti apache 459871 Oct 20 16:43 russian_russia.po
-rw-r--r--. 1 root  root   765568 Oct 20 16:43 spanish_spain.po
-rw-r--r--. 1 cacti apache 446032 Oct 20 16:43 swedish_sweden.po
```

Let me know if you need further info!

**Comment by cigamit** [comment]
Okay, status:

1 => As designed it is a comparison operator, so it's implied.  What do you think we should do here?
2 => Quite odd, code is correct, so not sure what is happening.
3 => fixed.
4 => Still researching
5 => @ronytomen has this on his plate along with the old non-vendor location cleanup

**Comment by netniV** [comment]
I would agree, 2 is weird given the lines below those that show %s are showing a value, unless for some reason it doesn't believe it has a value? Would it then replace the %s?

The example section should still show since you have directories without permissions.  That may be an issue of a null value somehow corrupting the string.  Did you have any errors in your apache or cacti log files?

**Comment by cigamit** [comment]
Issue 4 is due to the replacement text in the sprintf matching the default text domain of 'cacti'.  Since the 'cacti' text domain is implied, I've added an extra check during translations.  Maybe we need to simply make the default text domain something less common?  For now, I'm just adding the check, which will slow down page rendering a few milliseconds.

**Comment by cigamit** [comment]
@netniV, can you take a crack at 4?  @ronytomen is planning on releasing later this morning.

**Comment by netniV** [comment]
I've made a change for part 4, not sure if it'll work.  Unfortunately, I'm out for the night with the wife. so I won't be able to fully test its working.

On part 2, I am wondering if it believes that we are attempting to use an i18n domain with the value.  One solution would be to add an extra parameter to each call where the %s is showing up to see if that alone solves it.  Whilst the extra value wouldn't be put into the text, the __() function would not attempt to load it from the i18n domain as a result.

**Comment by jpobeda** [comment]
Sorry the delay, timezone.

> 1 => As designed it is a comparison operator, so it's implied. What do you think we should do here?

@cigamit ok, It looked odd to me the column without a name.

**Comment by jpobeda** [comment]
@netniV not sure, the string `Database User: <b>%s</b>` doesn't even get printed in Bold.

No apache or cacti errors. Since it's a test vm I could just reset the db and give it another go if you want to catch something in particular.

Let me know.

**Comment by cigamit** [comment]
It is in bold, just hard to see.  Can you confirm with latest codes that issue number 4 is resolved?  If so, we can close this.

**Comment by jpobeda** [comment]
I'll test it tomorrow and update

**Comment by netniV** [comment]
Looks like GitHub must have had a hiccup over night, a few  weird posts.
